Gruntfile.js

```diff
@@ -1,3 +1,4 @@
+// eslint-disable-next-line no-redeclare
```

Comment:
Needed to upgrade to eslint 7

Reply:
If need to apply the lint rule to the whole file, probably `/* eslint-disable no-redeclare */`

Also, looks like this happen in lots of files, how about moving it to eslint config file?

Reply:
Disabling `no-redeclare` globally could be risky as it can prevent detecting true lint errors.
As Aaron suggested, I've removed `/* global browser, ... */` comments except 2 files ( test/e2e/specs/basic_spec.js and test/e2e/specs/dev_spec.js )